app/full_text_search.py

```diff
@@ -114,7 +118,7 @@ def create_romanized_query_txt(query_txt):
     return enclose_in_parenthesis(romanized_query)
 
 
-def get_person_ids_from_results(romanized_query, results_list):
+def get_person_ids_from_results(romanized_query, results_list, returned_fields):
```

Comment:
Can you rename returned_fields to e.g., romanized_name_fields? "returned" doesn't make much sense in this method.